tests/integrations/go.mod

```diff
@@ -187,3 +207,5 @@ require (
 	gorm.io/driver/sqlite v1.4.3 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace github.com/pingcap/tidb-dashboard => github.com/lhy1024/tidb-dashboard v0.0.0-20240807103050-305fb37579af
```

Comment:
will removed after the dashboard update?

Reply:
yes, after https://github.com/pingcap/tidb-dashboard/pull/1707 merged.

Reply:
After discussion with @wuhuizuo, upgrade of etcd will be released in v8.4

Reply:
done